sympy/printing/pretty/pretty.py

```diff
@@ -1151,7 +1151,10 @@ def _print_SingularityFunction(self, e):
             pform = base**n
             return pform
         else:
-            return self._print_Function(e)
+            base = "<"
+            base += str(e.args[0]-e.args[1])
```

Comment:
using `base += self._print(e.args[0]-e.args[1])` gives `TypeError: Can't convert 'prettyForm' object to str implicitly`

Or did you mean the following 
```python
base += str(self._print(e.args[0]-e.args[1]))
````